src/Carbon/Lang/pt.php

```diff
@@ -28,6 +28,15 @@
     'from_now' => 'em :time',
     'after' => ':time depois',
     'before' => ':time antes',
+    'diff_now' => 'agora',
+    'diff_yesterday' => 'ontem',
+    'diff_tomorrow' => 'amanhã',
+    'diff_before_yesterday' => 'anteontem',
+    'diff_after_tomorrow' => 'depois de amanhã',
```

Comment:
Are you sure `anteontem` and mostly `depois de amanhã` are commonly used? If not, we should rather not translate them and so fallback to calendar translations.

Reply:
Thanks.